app/model/model.go

```diff
@@ -0,0 +1,33 @@
+package model
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/mizkei/self-lint-sample/app/data"
+	"github.com/mizkei/self-lint-sample/app/test"
+	atime "github.com/mizkei/self-lint-sample/app/time"
+)
+
+// Greet returns greeting message.
+func Greet(data data.Data) string {
+	now := atime.Now()
+
+	var msg string
+	switch {
+	case now.Hour() < 12:
+		msg = "good morning"
+	case now.Hour() < 18:
+		msg = "good afternoon"
+	default:
+		msg = "good evening"
+	}
+
+	return fmt.Sprintf("(%s), %s.", data.ID, msg)
+}
+
+// GreetIllegal is ...
+func GreetIllegal() string {
+	atime.SetTime(time.Date(2000, time.January, 1, 0, 0, 0, 0, time.Local))
```

Comment:
**[self-lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
forbidden to refer 'github.com/mizkei/self-lint-sample/app/time'.SetTime